### ISSUE TITLE
Errors while plotting using PyPlot

### ISSUE THREAD

**Issue by abhi18av** [JuliaPy/PyCall.jl#399]
Hello :)

I'm trying to use the PyPlot package, having installed the `juliaPro` distribution earlier.

But when I run the command for a simple plot
```julia

using PyPlot
x = linspace(0,2*pi,1000); y = sin(3*x + 4*cos(2*x))
plot(x, y, color="red", linewidth=2.0, linestyle="--")

```

I get the following error
```sh
QObject::connect: Cannot connect NavigationToolbar2QT::message(QString) to (null)::_show_message()
ERROR: PyError (:PyObject_Call) <class 'TypeError'>
TypeError('connect() failed between NavigationToolbar2QT.message[str] and _show_message()',)
  File "/Users/eklavya/anaconda/lib/python3.5/site-packages/matplotlib/pyplot.py", line 3147, in plot
    ax = gca()
  File "/Users/eklavya/anaconda/lib/python3.5/site-packages/matplotlib/pyplot.py", line 928, in gca
    return gcf().gca(**kwargs)
  File "/Users/eklavya/anaconda/lib/python3.5/site-packages/matplotlib/pyplot.py", line 578, in gcf
    return figure()
  File "/Users/eklavya/anaconda/lib/python3.5/site-packages/matplotlib/pyplot.py", line 527, in figure
    **kwargs)
  File "/Users/eklavya/anaconda/lib/python3.5/site-packages/matplotlib/backends/backend_qt5agg.py", line 43, in new_figure_manager
    return new_figure_manager_given_figure(num, thisFig)
  File "/Users/eklavya/anaconda/lib/python3.5/site-packages/matplotlib/backends/backend_qt5agg.py", line 51, in new_figure_manager_given_figure
    return FigureManagerQT(canvas, num)
  File "/Users/eklavya/anaconda/lib/python3.5/site-packages/matplotlib/backends/backend_qt5.py", line 465, in __init__
    self.toolbar.message.connect(self._show_message)

 in pyerr_check at /Applications/JuliaPro-0.5.2.1.app/Contents/Resources/pkgs-0.5.2.1/v0.5/PyCall/src/exception.jl:56 [inlined]
 in pyerr_check at /Applications/JuliaPro-0.5.2.1.app/Contents/Resources/pkgs-0.5.2.1/v0.5/PyCall/src/exception.jl:61 [inlined]
 in macro expansion at /Applications/JuliaPro-0.5.2.1.app/Contents/Resources/pkgs-0.5.2.1/v0.5/PyCall/src/exception.jl:81 [inlined]
 in #_pycall#66(::Array{Any,1}, ::Function, ::PyCall.PyObject, ::LinSpace{Float64}, ::Vararg{Any,N}) at /Applications/JuliaPro-0.5.2.1.app/Contents/Resources/pkgs-0.5.2.1/v0.5/PyCall/src/PyCall.jl:554
 in (::PyCall.#kw##_pycall)(::Array{Any,1}, ::PyCall.#_pycall, ::PyCall.PyObject, ::LinSpace{Float64}, ::Vararg{Any,N}) at ./<missing>:0
 in #pycall#70(::Array{Any,1}, ::Function, ::PyCall.PyObject, ::Type{PyCall.PyAny}, ::LinSpace{Float64}, ::Vararg{Any,N}) at /Applications/JuliaPro-0.5.2.1.app/Contents/Resources/pkgs-0.5.2.1/v0.5/PyCall/src/PyCall.jl:571
 in (::PyCall.#kw##pycall)(::Array{Any,1}, ::PyCall.#pycall, ::PyCall.PyObject, ::Type{PyCall.PyAny}, ::LinSpace{Float64}, ::Vararg{Any,N}) at ./<missing>:0
 in #call#71(::Array{Any,1}, ::PyCall.PyObject, ::LinSpace{Float64}, ::Vararg{Any,N}) at /Applications/JuliaPro-0.5.2.1.app/Contents/Resources/pkgs-0.5.2.1/v0.5/PyCall/src/PyCall.jl:574
 in (::PyCall.#kw#PyObject)(::Array{Any,1}, ::PyCall.PyObject, ::LinSpace{Float64}, ::Vararg{Any,N}) at ./<missing>:0



```

I've made sure that the `Python` env variable is pointed to the Anaconda distribution

```julia
julia> using PyCall

julia> @pyimport sys as sys

julia> print(sys.version)
3.5.2 |Anaconda custom (x86_64)| (default, Jul  2 2016, 17:52:12) 
[GCC 4.2.1 Compatible Apple LLVM 4.2 (clang-425.0.28)]

```

How can I get this to work fine?

**Comment by stevengj** [comment]
(Normally you would report an issue like this in PyPlot.jl, not PyCall.jl.)

Apparently this was a Matplotlib bug, fixed in Matplotlib 1.5.3: matplotlib/matplotlib#6858 ... you can try using a different Matplotlib backend in the meantime, e.g. TkAgg.

**Comment by abhi18av** [comment]
Yes, upgrading matplotlib solved the issue, thanks @stevengj .

Actually, I thought that It ultimately depends upon the `PyCall` package that perhaps, I should file it here.  But, I'll keep what you said in mind, for future reference.
